.gitignore

```diff
@@ -20,3 +20,5 @@
 !.gitattributes
 !map.js
 !index.js
+!*.swp
+!*.swo
```

Comment:
Wait, I don't think this does what you think it does.

This is an ignore file.  And these are reverse ignores.  This patch is causing them to be _included_ by git, not ignored.